libs/components/behavioral/src/lib/biomarker-details/biomarker-details.component.scss

```diff
@@ -0,0 +1,85 @@
+:host {
+  display: grid;
+  grid-template-columns: 21.5rem 21.5rem auto;
+  grid-template-rows: 3.5rem auto 3rem 5rem 3.5rem auto;
+
+  .header-label {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    grid-column: 1 / 4;
+    grid-row: 1;
+    padding: 0.25rem 2rem;
+    gap: 0.5rem;
+  }
+
+  .table {
+    grid-column: 1 / 4;
```

Comment:
Consider replacing grid-column/grid-row with grid-area and add a grid-template-area on the :host level for readability
https://css-tricks.com/snippets/css/complete-guide-grid/#aa-grid-template-areas